pykokkos/lib/ufunc_workunits.py

```diff
@@ -1,6 +1,563 @@
 import pykokkos as pk
 
 
+@pk.workunit
+def equal_impl_5d_int8(tid: int,
+                              view1: pk.View5D[pk.int8],
+                              view2: pk.View5D[pk.int8],
+                              out: pk.View5D[pk.uint8]):
```

Comment:
as with several recent PRs, feel free to help with formatting for the large blocks of largely copy-paste templates of workunits..